CMakeLists.txt

```diff
@@ -531,6 +533,11 @@ else()
     option(FILAMENT_DISABLE_MATOPT "Disable material optimizations" ON)
 endif()
 
+# Build Filament with multiview extension for stereo rendering. This disables the instanced stereo implementation.
+if (FILAMENT_USE_MULTIVIEW_STEREO)
+  add_definitions(-DFILAMENT_USE_MULTIVIEW_STEREO)
```

Comment:
I don't think we need to add a C++ define for this. Filament itself shouldn't need to know which type of stereo our samples use.